pkgs/development/libraries/flatpak/fix-test-paths.patch

```diff
@@ -9,16 +9,7 @@
          sed -e 's/^/# /' < bwrap-result
          echo "1..0 # SKIP Cannot run bwrap"
          exit 0
-@@ -323,7 +323,7 @@
```

Comment:
Why were these fragments removed?

Reply:
The patches didn't apply, in particular the `skip_without_python2` function is gone. Probably should've left an explanation in the commit message. Looks like there's no python2 support anymore.